HelpersWayland.cpp

```diff
@@ -0,0 +1,295 @@
+#ifdef WAYLAND_PROTOCOL
+
+#include <iostream>
+
+#include "HelpersWayland.h"
+
+bool HelpersWayland::hasProperLibEGL() {
+    driverName = (eglGetDisplayDriverName_t *) eglGetProcAddress("eglGetDisplayDriverName");
+    driverConfig = (eglGetDisplayDriverconfig_t *) eglGetProcAddress("eglGetDisplayDriverConfig");
+
+    if (!driverName || !driverConfig) {
+        std::cerr << "Error getting function pointers. LibEGL must be too old.\n";
```

Comment:
Add this string to the translations using the proper macro `_("Error getting function pointers. LibEGL must be too old.\n")`